product_variant_search_by_attribute/models/attribute.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+# © 2016 David BEAL @ Akretion <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import api, models
+
+
+_logger = logging.getLogger(__name__)
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = 'product.attribute.value'
+
+    @api.multi
+    def write(self, vals):
+        """ Attribute Values update impact products: we take care
+            to propagate changes on attribute_str field"""
+        res = super(ProductAttributeValue, self).write(vals)
+        if 'name' in vals:
+            sql = """SELECT prod_id
```

Comment:
Anyway, why not do this through ORM? We should avoid SQL the most we are able to.